conquest-token/pom.xml

```diff
@@ -54,6 +54,10 @@
             <groupId>com.networknt</groupId>
             <artifactId>http-client</artifactId>
         </dependency>
+        <dependency>
```

Comment:
client-config is in the pom.xml already. This is the same for other pom.xml updates.